composer.json

```diff
@@ -35,7 +35,7 @@
         "drupal/views_add_button_group": "^2.0",
         "fontawesome/fontawesome": "5.15.4",
         "drupal/twig_field_value": "^2.0",
-        "bootstrap/bootstrap": "^5.1.0",
+        "bootstrap/bootstrap": "4.3.1",
```

Comment:
Was this the earliest working version of Bootstrap to revert to? Not a fan of going back that far if we can help it.

Reply:
I'm double-checking now to see if there are other versions b/w 4.3.1 and 5.1.0 that do not re-introduce the issue.